homeassistant/components/mqtt/alarm_control_panel.py

```diff
@@ -11,6 +11,7 @@
     SUPPORT_ALARM_ARM_CUSTOM_BYPASS,
     SUPPORT_ALARM_ARM_HOME,
     SUPPORT_ALARM_ARM_NIGHT,
+    SUPPORT_ALARM_ARM_VACATION,
```

Comment:
Please don't change any platforms in this PR. That should be done in separate PRs after this is merged.

Reply:
So mqtt and manual_mqtt changes need to be reverted and put in later after merge?

Reply:
Yes, please. 👍